util/clientmetric/clientmetric.go

```diff
@@ -22,12 +22,27 @@ import (
 var (
 	mu          sync.Mutex // guards vars in this block
 	metrics     = map[string]*Metric{}
-	numWireID   int       // how many wireIDs have been allocated
-	lastDelta   time.Time // time of last call to EncodeLogTailMetricsDelta
-	sortedDirty bool      // whether sorted needs to be rebuilt
-	sorted      []*Metric // by name
+	numWireID   int         // how many wireIDs have been allocated
+	lastDelta   time.Time   // time of last call to EncodeLogTailMetricsDelta
+	sortedDirty bool        // whether sorted needs to be rebuilt
+	sorted      []*Metric   // by name
+	lastLogVal  []scanEntry // by Metric.regIdx
+	unsorted    []*Metric   // by Metric.regIdx
+
+	// valBackBuf is a set of free contiguous int64s to assign to Metric.v
```

Comment:
I suggest calling this `freelist` or `int64alloc` or the like. This makes it clearer that the purpose is a custom int64 allocator.

"to assign to" isn't technically quite right--we're taking the address, not just assigning.

Reply:
done